site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2248,10 +2276,22 @@ public function getTotalComponentCount($g_id) {
         return intval($this->course_db->row()['cnt']);
     }
 
+    /**
+     * Returns the total number of verified components found in the database
+     *
+     * @param  string $g_id
+     * @return int
+     */
+    public function getTotalVerifiedComponentCount(string $g_id): int {
+        $this->course_db->query("SELECT count(*) AS cnt FROM gradeable_component_data WHERE GCD_VERIFIER_ID IS NOT NULL", [$g_id]);
```

Comment:
You are passing a SQL parameter here, but there aren't any placeholders in the SQL query.